src/orm/product.ts

```diff
@@ -0,0 +1,79 @@
+import type { Sequelize } from 'sequelize';
+import { DataTypes } from 'sequelize';
+import { initModel, makeOneToMany, Model } from '@padoa/database';
+import { Team } from '@padoa/meta';
+import { ProductOrigin, ProductPhysicalState, ProductWarningNotice } from '@padoa/chemical-risk';
+
+import { ModelName } from '@src/orm/model.js';
+
+class Product extends Model {
+  public declare fdsFileId?: number;
+  public declare name?: string;
+  public declare producer?: string;
+  public declare origin?: ProductOrigin;
+  public declare physicalState?: ProductPhysicalState;
+  public declare boilingPoint?: string;
+  public declare vaporPressure?: string;
+  public declare warningNotice?: ProductWarningNotice;
+  public declare revisionDate?: Date;
+}
+
+const initProduct = (sequelize: Sequelize): void => {
+  initModel(
+    Product,
+    {
+      name: {
+        type: DataTypes.STRING,
+        allowNull: true,
+      },
+      producer: {
+        type: DataTypes.STRING,
+        allowNull: true,
+      },
+      origin: {
+        type: DataTypes.ENUM,
+        values: Object.values(ProductOrigin),
+        allowNull: true,
```

Comment:
```suggestion
        allowNull: false,
``` c.f migrations ?(voir si on le fait)